src/Uno.UWP/ApplicationModel/Resources/ResourceLoader.cs

```diff
@@ -48,6 +48,15 @@ public ResourceLoader()
 
 		public string GetString(string resource)
 		{
+			// "/[file]/[name]" format support
+			if (resource.ElementAtOrDefault(0) == '/')
+			{
+				var separatorIndex = resource.IndexOf("/", 1);
```

Comment:
Thanks for the modifications!

There should be a validation of the format though (and a test that validates that as well).

For instance, if the provided string is `/myfile` the substring will fail with an exception. With UWP, such a lookup returnns an empty string, so this will do:
```suggestion
				var separatorIndex = resource.IndexOf("/", 1);
				
				if(separatorIndex != null)
				{
```

With an additional test that validates that `/this_is_not_a_valid_file` returns `""`

Reply:
Very good point...
But why null (`!= null`)? It cannot be null (=0), 'not found' is -1 :)

Reply:
File not found - UWP returns "",
This test (for 'file exist') should be in GetForCurrentView(resourceFile) method, so it is something outside my PR (and I'm lost in code flow).